src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java

```diff
@@ -111,6 +112,7 @@ public ResponseEntity<String> putUserCommonsByIdSell(
 
         if(userCommons.getNumOfCows() >= 1 ){
           userCommons.setTotalWealth(userCommons.getTotalWealth() + commons.getCowPrice());
+          userCommons.setTotalCowHealth(userCommons.getTotalCowHealth() - (userCommons.getTotalCowHealth() / userCommons.getNumOfCows()));
```

Comment:
This formula is (a) incorrect (see detailed comments on code review) (b) should be factored out into a method, so that we aren't repeating the formula from line 90.